util/raster_mosaic.py

```diff
@@ -63,19 +33,17 @@ def image_grid_overlap(image_list, rows, cols, crop):
     off_y = 0
 
     # Creates new empty image with taking the size from sub-tile from the list
-
     for i, img in enumerate(image_list):
         img = Image.open(img)
 
         border = (crop, crop, crop, crop)  # left, up, right, bottom
 
         # Get row and column number
-        col_id = i % cols
-        row_id = i // cols
+        col_id = i // cols
+        row_id = i % cols
```

Comment:
Would be nice to check if it always INT

Reply:
I'm pretty sure here it's INT, because when I was writing this function I was checking what is what and I do not remember it to be float